packages/contracts-bedrock/src/dispute/FaultDisputeGameN.sol

```diff
@@ -887,10 +911,11 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         // If the move is a defense, the disputed output could have been made by either party. In this case, we
         // need to search for the parent output to determine what the expected status byte should be.
         Position disputedLeafPos = Position.wrap(_parentPos.raw() + 1);
```

Comment:
I believe this line is incorrect for Nary > 2 cases - the purpose is to find the position of the post claim hash and thus it should be _parentPos.raw() + _attackBranch.